src/Display/Slider/Slider.stories.tsx

```diff
@@ -67,3 +80,59 @@ RemoveDots.args = { removeDots: true };
 
 export const InitialItem = Template.bind({});
 InitialItem.args = { initialItem: 2 };
+
+export const CustomizedRenderLeftIcon: Story<Props> = () => (
```

Comment:
Can't you do
```
export const InitialItem = Template.bind({});
CustomizedRenderLeftIcon.args = { 
    renderLeftIcon={(disabled: boolean) => {
      const color = disabled ? Greyscale.grey : SecondaryColor.actionblue;
      return <ArrowBackDoubleIcon color={color} />;
    }}
};
```
here?

Reply:
I had done this initially but this function wasn't visible in the source code 
The source code becomes like
<Slider renderLeftIcon={() => {}}>
..........
</Slider>

Reply:
Ooh I see, then this is okay.